python/cudf/cudf/tests/test_transform.py

```diff
@@ -27,3 +27,22 @@ def generic_function(a):
     result = lhs_arr ** 3
 
     np.testing.assert_almost_equal(result, out_col.to_array())
+
+
+@pytest.mark.parametrize("dtype", supported_types)
+def test_applymap_python_lambda(dtype):
+
+    size = 500
+
+    lhs_arr = np.random.random(size).astype(dtype)
+    lhs_col = Series(lhs_arr)._column
+
+    # Note that the lambda has to be written this way.
+    # In other words, the following code does NOT compile with numba:
+    # test_list = [1, 2, 3, 4]
+    # out_col = lhs_col.applymap(lambda x: x in test_list)
+    out_col = lhs_col.applymap(lambda x: x in [1, 2, 3, 4])
+
+    result = np.isin(lhs_arr, [1, 2, 3, 4])
+
+    np.testing.assert_almost_equal(result, out_col.to_array())
```

Comment:
```suggestion
    lhs_ser = Series(lhs_arr)

    # Note that the lambda has to be written this way.
    # In other words, the following code does NOT compile with numba:
    # test_list = [1, 2, 3, 4]
    # out_ser = lhs_ser.applymap(lambda x: x in test_list)
    out_ser = lhs_ser.applymap(lambda x: x in [1, 2, 3, 4])

    result = np.isin(lhs_arr, [1, 2, 3, 4])

    np.testing.assert_almost_equal(result, out_ser.to_array())
```

Should we rather have this pytest on series object?